CHANGELOG.md

```diff
@@ -184,6 +186,10 @@ Reflection
     the telescope types, the contents of a `dot` pattern can safely be
     set to `unknown`.
 
+- New operation in `TC` monad, `execTC`, which calls an external executable
+  ```agda
+  execTC : String → List String → String → TC (Σ Nat (λ _ → Σ String (λ _ → String)))
```

Comment:
Explain  purpose of these `String`s.

Reply:
Added a comment regarding the inputs and outputs, and regarding the `--allow-exec` option and the `~/.agda/executables` file.